.eslintrc.json

```diff
@@ -0,0 +1,30 @@
+{
+  "env": {
+    "browser": true,
+    "es6": true,
+    "jest": true,
+    "node": true
+  },
+  "extends": [
+    "eslint:recommended",
+    "plugin:react/recommended"
+  ],
+  "parserOptions": {
+    "ecmaFeatures": {
+      "jsx": true
+    },
+    "ecmaVersion": 11,
+    "sourceType": "module"
+  },
+  "plugins": [
+    "react"
+  ],
+  "rules": {
```

Comment:
Let me know what you think about these rules, I feel they are basic enough to keep our code more consistent (I noticed a lot of inconsistency in the codebase) without being too pushy.

Reply:
I'll have to play around with it to tell you what I think - but mostly, just having consistent code styling is the important part. Thanks for doing this